exonum-java-binding/core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -49,27 +42,14 @@ pub struct JavaServiceRuntime {
 }
 
 impl JavaServiceRuntime {
-    /// Creates new runtime from provided config or returns the one created earlier.
-    ///
-    /// There can be only one `JavaServiceRuntime` instance at a time.
-    pub fn get_or_create(config: Config) -> Self {
-        unsafe {
-            // Initialize runtime if it wasn't created before.
-            JAVA_SERVICE_RUNTIME_INIT.call_once(|| {
-                let java_vm = Self::create_java_vm(&config.jvm_config, &config.runtime_config);
-                let executor = MainExecutor::new(Arc::new(java_vm));
-                let service_runtime =
-                    Self::create_service_runtime(config.runtime_config, executor.clone());
-                let runtime = JavaServiceRuntime {
-                    executor,
-                    service_runtime,
-                };
-                JAVA_SERVICE_RUNTIME = Some(runtime);
-            });
-            // Return global runtime.
-            JAVA_SERVICE_RUNTIME
-                .clone()
-                .expect("Trying to return runtime, but it's uninitialized")
+    /// Creates new runtime from provided config.
+    pub fn create_java_runtime(config: Config) -> Self {
+        let java_vm = Self::create_java_vm(&config.jvm_config, &config.runtime_config);
+        let executor = MainExecutor::new(Arc::new(java_vm));
+        let service_runtime = Self::create_service_runtime(config.runtime_config, executor.clone());
```

Comment:
The difference between
`create_java_runtime`
and
`create_service_runtime` 
is not clear. I'd think of better names (swap them?).

Reply:
I think this method actually needs to be called `new` (current JavaServiceRuntime::create_java_runtime)